src/ppx/grfx/vk/vk_device.cpp

```diff
@@ -236,8 +241,20 @@ Result Device::ConfigureFeatures(const grfx::DeviceCreateInfo* pCreateInfo, VkPh
 {
     vk::Gpu* pGpu = ToApi(pCreateInfo->pGpu);
 
-    VkPhysicalDeviceFeatures foundFeatures = {};
-    vkGetPhysicalDeviceFeatures(pGpu->GetVkGpu(), &foundFeatures);
+    VkPhysicalDeviceFeatures          foundFeatures          = {};
+    VkPhysicalDeviceMultiviewFeatures foundMultiViewFeatures = {VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_MULTIVIEW_FEATURES};
```

Comment:
This is not used anywhere. I think in theory we need to check this in line 616 because in theory this can be false.